engine/apps/grafana_plugin/tests/test_app_config.py

```diff
@@ -2,7 +2,7 @@
 from unittest.mock import patch
 
 import pytest
-from django.apps import apps
+from django.apps import apps  # noqa: I251
```

Comment:
This is the only place in the entire repo using `noqa: I251`, because `django.apps.apps` is used for something different than `get_model` here.